addons/mrp/models/mrp_production.py

```diff
@@ -247,6 +246,18 @@ def _get_default_is_locked(self):
     show_produce = fields.Boolean(compute='_compute_show_produce', help='Technical field to check if produce button can be shown')
     show_produce_all = fields.Boolean(compute='_compute_show_produce', help='Technical field to check if produce all button can be shown')
     is_outdated_bom = fields.Boolean("Outdated BoM", help="The BoM has been updated since creation of the MO")
+    product_template_attribute_value_ids = fields.Many2many(
+        string="product template attribute values",
+        related='product_id.product_template_attribute_value_ids',
+        depends=['product_id'])
+    is_configurable_product = fields.Boolean(
+        string="Is the product configurable?",
+        related='product_id.product_tmpl_id.has_configurable_attributes',
+        depends=['product_id'])
+    description = fields.Text(
+        string="Description",
+        compute='_compute_description',
+        store=True, readonly=False, required=True, precompute=True)
```

Comment:
why isn't all this in the mixin then ?

Reply:
So the problem we are facing is that the related fields are not related to the same fields. For instance : 
- in ``sale_order_line`` => ```is_configurable_product = fields.Boolean(
        string="Is the product configurable?",
        related='product_template_id.has_configurable_attributes',
        depends=['product_id'])```
- in ``mrp_production`` => ```    is_configurable_product = fields.Boolean(
        string="Is the product configurable?",
        related='product_id.product_tmpl_id.has_configurable_attributes',
        depends=['product_id'])```

In a similar way, the description field is not common to ``mrp_production and`` to ``sale_order_line`` as ``sale_order_line`` use the ``name`` field (which is reserved to the manufactoring name in ``mrp_production``